src/client/writer.rs

```diff
@@ -266,6 +266,10 @@ fn content_encoder(buf: BytesMut, req: &mut ClientRequest) -> Output {
             }
             #[cfg(not(any(feature = "flate2", feature = "brotli")))]
             {
+                let mut b = BytesMut::new();
+                let _ = write!(b, "{}", bytes.len());
+                req.headers_mut()
+                    .insert(CONTENT_LENGTH, HeaderValue::try_from(b.freeze()).unwrap());
```

Comment:
Is there a reason not to use `bytes.len().to_string()`?

Reply:
not sure, I've just copied what was there previously

Reply:
ah ok then